packages/tools/buildServerDocker.ts

```diff
@@ -27,7 +37,8 @@ async function main() {
 	} catch (error) {
 		console.info('Could not get git commit: metadata revision field will be empty');
 	}
-	const buildArgs = `--build-arg BUILD_DATE="${buildDate}" --build-arg REVISION="${revision}" --build-arg VERSION="${imageVersion}"`;
+	const buildArgs = { BUILD_DATE: buildDate, REVISION: revision, VERSION: imageVersion };
+	const flattenedBuildArgs = flatten((Object.keys(buildArgs) as (keyof typeof buildArgs)[]).map((k) => ['--build-arg', `${k}=${buildArgs[k]}`]));
```

Comment:
I'd rather we keep it simple even if it means more lines of code. So since those are command lines args, you can do this:

```
const buildArgs = [
   '--build-arg BUILD_DATE="${buildDate}"',
   '--build-arg REVISION="${revision}"',
   '--build-arg VERSION="${imageVersion}"',
]
```

Also no need for both a buildArgs and flattenedBuildArgs variable.